examples/dreambooth/train_dreambooth_lora.py

```diff
@@ -839,6 +839,11 @@ def main(args):
         else:
             raise ValueError("xformers is not available. Make sure it is installed correctly")
 
+    if args.gradient_checkpointing:
+        unet.enable_gradient_checkpointing()
```

Comment:
for LoRA gradients still flow through the unet so we should enable gradient checkpointing